packaging/darwin/amazon-cloudwatch-agent-ctl

```diff
@@ -167,23 +167,6 @@ cwa_config() {
 
      mkdir -p "${CONFDIR}"
 
-     param_mode="ec2"
```

Comment:
As of right now, I think that this should not be removed from the ctl script, although I don think that the default mode should be `auto`, not `ec2`